docs/software/programming.md

```diff
@@ -67,4 +68,25 @@ Source: [User:`Maklaan` - Based on a Docker blog post](https://commons.wikimedia
 
 See also <https://wikipedia.org/wiki/Docker_(software)> resp. <https://docs.docker.com/get-started/overview>.
 
+## Portainer
+
+Portainer simplifies your Docker container management via Portainer web interface. It enables faster deploy of the applications and it gives real time visibility.
+
+![DietPi Optimised Software Portainer](../assets/images/dietpi-software-portainer.jpg)
+
+=== "Quick start"
+
+    - Start Portainer using next url in the browser (`https://<your.local.ip>:9002` or `https://<your.host.name>:9002`)[^1].
+
+    - If you have basic questions about how to use Portainer, use the next beginner guide to Portainer: <https://codeopolis.com/posts/beginners-guide-to-portainer/>.
+
+=== "Official documentation"
+
+    - Documentation: <https://documentation.portainer.io/>
+    - Source code: <https://github.com/portainer/portainer>
+    - Open-source license: zlib
+
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+[^1]:
+    [Logitech Media Server](../media/#logitech-media-server) already listened to port `9000`, and this is why **Portainer** has been configured to start using port `9002`. Container internal port needs to stay `9000`, and the actual command is `docker run -d -p 9002:9000`. For more details on the implementation Portainer in DietPi see the GitHub task: <https://github.com/MichaIng/DietPi/pull/3933>
```

Comment:
by default Portainer is setup to automatically start. No need to start it manually.

Reply:
fixed in https://github.com/MichaIng/DietPi-Docs/pull/284